plugins/hydra_ax_sweeper/tests/test_ax_sweeper_plugin.py

```diff
@@ -212,17 +251,23 @@ def test_example_app(tmpdir: Path) -> None:
         "example/banana.py",
         "-m",
         "hydra.run.dir=" + str(tmpdir),
-        "banana.x=-5:5",
-        "banana.y=-5:10.1",
+        "banana.x=int(interval(-5, 5))",
+        "banana.y=interval(-5, 10.1)",
         "hydra.sweeper.ax_config.max_trials=2",
     ]
     result = subprocess.check_output(cmd).decode("utf-8").rstrip()
-    assert "banana.x: range=[-5, 5], type = int" in result
-    assert "banana.y: range=[-5.0, 10.1], type = float" in result
+    assert "banana.x: range=[-5, 5]" in result
+    assert "banana.y: range=[-5.0, 10.1]" in result
 
 
-@pytest.mark.parametrize("overrides", [[], ["quadratic.x_arg=-1:1"]])
-def test_jobs_configured_via_nested_config(hydra_sweep_runner, overrides: list) -> None:
+@pytest.mark.parametrize(
+    "overrides",
+    [[], ["quadratic.x_arg=int(interval(-1, 1))"]],
+)
+def test_jobs_configured_via_nested_config(
+    hydra_sweep_runner,
+    overrides: list,
```

Comment:
remove trailing comma like the one after list. black will format in a more compact way.

Reply:
ahh nice!